src/components/VMapboxStylePicker.vue

```diff
@@ -41,19 +54,37 @@ export default {
       // initialize control
       this.map.addControl(this, 'bottom-right')
 
+      // creating the variable
+      var year = 2016
+
+      // moment and evetbus connection
+      bus.$on('slider-update', (date) => {
+        var b = date.enddate.slice(3)
+        const enddate = moment(b).format('YYYY')
+        if (enddate <= '2016') {
```

Comment:
reuse the variable you made for 'year' -> if need to change you only have to do it in 1 location. 
In this line, but also the next line -> you now define '2016' in 3 locations

Always think of how you can make the code as short as possible. 
Something like this: 

let year = 2016
if (enddate >year) {
  year = enddate
}